tools/integration/test/integration/harvestTest.js

```diff
@@ -10,7 +10,9 @@ describe('Tests for harvesting different components', function () {
   it('should verify all harvests are complete', async function () {
     this.timeout(harvest.timeout)
     console.time('Harvest Test')
-    const status = await harvestTillCompletion(components)
+    const recentDefinitions = await getComponents()
+    console.info(`Recent definitions: ${recentDefinitions}`)
+    const status = await harvestTillCompletion(recentDefinitions)
```

Comment:
```suggestion
    const targetDefinitions = await getComponents()
    console.info(`Recent definitions: ${targetDefinitions}`)
    const status = await harvestTillCompletion(targetDefinitions)
```
Didn't check to see if the constant is used beyond these lines.